src/app.js

```diff
@@ -1,21 +1,36 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const fs = require('fs');
+
+const renameFile = (oldName, newName, isExist) => {
+  const oldPath = `src/${oldName}`;
+  const newPath = isExist ? `src/${newName}/${oldName}` : `src/${newName}`;
+
+  fs.rename(oldPath, newPath, (err) => {
+    if (err) {
+      throw new Error('Rename error');
+    }
+  });
+};
+
+const app = () => {
+  const args = process.argv.slice(2);
+  const firstPath = args[0];
+  const secondPath = args[1];
+
+  try {
+    const isExists = fs.existsSync(`src/${secondPath}`);
+
+    if (secondPath[secondPath.length - 1] === '/' && !isExists) {
+      throw new Error('No such directory');
```

Comment:
The directory check should be done for all directories, not just the ones ending with '/'. Please, consider changing the if condition.

Reply:
It actually was done for this
``node index file.txt dir/` moves the file under the `dir`. In case `dir` does not exist an error is thrown.`
All other cases work as they should